cmd/p4info_code_gen/p4info_code_gen.go

```diff
@@ -7,13 +7,15 @@ import (
 	"encoding/binary"
 	"flag"
 	"fmt"
-	log "github.com/sirupsen/logrus"
 	"os"
 	"sort"
 	"strings"
 
 	"github.com/ettle/strcase"
 	"github.com/golang/protobuf/proto"
```

Comment:
Given that this package is deprecated, should we move to use `google.golang.org/protobuf/proto` instead?
This could be done as part of this PR and in another PR.

```suggestion
	"google.golang.org/protobuf/proto"
```